### PR TITLE
Refactored GLTFWrapper to Runtime namespace

### PR DESCRIPTION
**Pull request by kcoley**
This PR converts the GLTFWrapper code into a Runtime namespace equivalent.  

- The classes have been separated into their own files
- The methods and properties now begin with capital letters
- GLTFWrapper has been refactored to Runtime.GLTF in the solution
- Unit tests have been moved to Source directory
- Unit tests have been updated to use Runtime code
- Program.cs, TestValues.cs and Common.cs have been updated to use Runtime code.

### CODE REVIEW

**Comment by kcoley** [comment]
I back-imported the recent base color factor alpha change on the develop branch

File: Source/Program.cs [hunk 1]
```diff
@@ -3,7 +3,7 @@
 using System;
 using System.IO;
 using System.Reflection;
-using static AssetGenerator.GLTFWrapper;
+//using static AssetGenerator.GLTFWrapper;
 using System.Diagnostics;
```
Comment (stevk) [inline Source/Program.cs]: Delete `using static AssetGenerator.GLTFWrapper;` instead of commenting it out?

Comment (kcoley) [reply Source/Program.cs]: Good catch. I'll remove that line.

**Comment by kcoley** [comment]
This last update changes out the glTFLoader NuGet to the newly added glTF2Loader NuGet package.

**Pull request merged**